android/java/res/drawable/rewards_tipping_radio_group_bg.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+	<solid android:color="@color/rewards_tipping_radio_group_bg"/>
+    <corners android:radius="20dp"/>
+</shape>
```

Comment:
Add new line at the end as per chromium guidelines

Reply:
done